src/Lachain.Core/Blockchain/SystemContracts/GovernanceContract.cs

```diff
@@ -269,11 +281,11 @@ public ExecutionStatus DistributeCycleRewardsAndPenalties(UInt256 cycle, SystemC
         }
 
         [ContractMethod(GovernanceInterface.MethodKeygenConfirm)]
-        public ExecutionStatus KeyGenConfirm(UInt256 cycle, byte[] tpkePublicKey, byte[][] thresholdSignaturePublicKeys,
+        public ExecutionStatus KeyGenConfirm(UInt256 cycle, byte[] tpkePublicKey, byte[][] tpkeVerificationKeys, byte[][] thresholdSignaturePublicKeys,
```

Comment:
node without hardfork 12 will send 3 parameters with tx. node with hardfork 12 will expect 4 parameters. This could cause error. Maybe we can set a default value for tpkeVerificationKeys ?